app/org/sagebionetworks/bridge/models/accounts/User.java

```diff
@@ -176,11 +153,35 @@ public boolean isInRole(Roles role) {
     public boolean isInRole(Set<Roles> roleSet) {
         return roleSet != null && !Collections.disjoint(roles, roleSet);
     }
+    
+    public List<ConsentStatus> getConsentStatuses() {
+        return consentStatuses;
+    }
+    
+    public void setConsentStatuses(List<ConsentStatus> consentStatuses) {
+        this.consentStatuses = (consentStatuses == null) ? ImmutableList.of() : ImmutableList.copyOf(consentStatuses);
+    }
```

Comment:
Should be okay, but I am going to leave the guid in the object as well to make it easier for clients to iterate over. I image they are basically going to iterate through them, taking action on each one.